e2e/app/component/cell.ts

```diff
@@ -0,0 +1,14 @@
+import Container from 'app/container';
+import { Page } from 'puppeteer';
+import { getPropValue } from 'utils/element-utils';
+
+export default class Cell extends Container {
```

Comment:
I plan to make a follow up PR to move `table.ts` and `cell.ts` from `component` dir to new `table` dir. At some point in future, new `column.ts` will be added.